### ISSUE TITLE
Feature Request - Full product name on the products list

### ISSUE THREAD

**Issue by futurehousestore** [Dolibarr/dolibarr#8379]
# Feature Request
I think the full product name on the products list would be very helpful for many users not only me. At the moment product name is restricted and longer names are shorten by "..." even if there is enough space to fit full product name.

<img width="1442" alt="screen shot 2018-03-13 at 21 33 02" src="https://user-images.githubusercontent.com/28427270/37371400-2483ae50-2707-11e8-9a3c-b32a304a2add.png">

## Use case
That would speed up product search if we don't remember the full name or code of the product.

## Suggested implementation
responsive product names?

**Comment by hurricane66750** [comment]
it's a good proposal !!

**Comment by futurehousestore** [comment]
I found a workaround myself to do it. In `htdocs/products/list.php` I have changed the value 40 to 70 (All my product names are below 70 characters) and I have changed `class="tdoverflowmax200` to `class="tdoverflowmax220`
so the code now looks like this 
```
// Label
		if (! empty($arrayfields['p.label']['checked']))
		{
			print '<td class="tdoverflowmax220">'.dol_trunc($obj->label, 70).'</td>';
			if (! $i) $totalarray['nbfield']++;
		}
```

below is the code to display full names + code to add product pictures on the product list :)
```
// Ref
		if (! empty($arrayfields['p.ref']['checked']))
		{
			print '<td class="tdoverflowmax30">';
			print $product_static->getNomUrl(1);
			
                        $tmpphoto = $product_static->show_photos('product', $conf->product->multidir_output[$product_static->entity], 1, 1, 0, 0, 0, 40);
                         if ($product_static->nbphoto > 0) { 
				print '<br>'.$tmpphoto;
                          }

			print "</td>\n";
			print "</td>\n";
			if (! $i) $totalarray['nbfield']++;
		}
		// Ref supplier
		if (! empty($arrayfields['pfp.ref_fourn']['checked']))
		{
			print '<td class="tdoverflowmax200">';
			print $product_static->getNomUrl(1);
			print "</td>\n";
			if (! $i) $totalarray['nbfield']++;
		}
		// Label
		if (! empty($arrayfields['p.label']['checked']))
		{
			print '<td class="tdoverflowmax220">'.dol_trunc($obj->label, 70).'</td>';
			if (! $i) $totalarray['nbfield']++;
		}

		// Type
```

**Comment by futurehousestore** [comment]
Fixed with #10732
